Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp

```diff
@@ -547,6 +544,7 @@ namespace LegacyLevelSystem
 
         OnUnloadComplete(m_lastLevelName.c_str());
 
+        // Delete level entities and prefab instances and remove them from the game entity context
```

Comment:
In game mode, the editor entities and prefabs would be converted to flattened list of entities under the root spawnable. So, its better to avoid referring to prefabs whenever possible. I suggest changing this to "Delete level entities and remove..."

Reply:
Makes sense. I will change that comment.